glmnet/util.py

```diff
@@ -13,8 +13,9 @@
 from .scorer import check_scoring
 
 
-def _score_lambda_path(est, X, y, sample_weight, relative_penalties, cv, scoring, classifier,
-                       n_jobs, verbose):
+def _score_lambda_path(est, X, y, sample_weight, relative_penalties,
+                       n_splits, random_state, scoring, n_jobs,
+                       verbose):
```

Comment:
I think you can now remove `n_splits` and `random_state` now that cv is handled by the estimator.

Reply:
Fixed